hack/prepare-supervisor-on-kind.sh

```diff
@@ -446,16 +544,16 @@ fi
 # they expire, so you should not be prompted to log in again for the rest of the day.
 if [[ "$use_oidc_upstream" == "yes" ]]; then
   echo "To log in using OIDC, run:"
-  echo "PINNIPED_DEBUG=true https_proxy=\"$PINNIPED_TEST_PROXY\" no_proxy=\"127.0.0.1\" ./pinniped whoami --kubeconfig ./kubeconfig-oidc.yaml"
+  echo "PINNIPED_DEBUG=true ${proxy_env_vars}./pinniped whoami --kubeconfig ./kubeconfig-oidc.yaml"
```

Comment:
Does this need a trailing space after `${proxy_env_vars}` before `./pinniped`?

Reply:
Good question. There is purposefully no space there. See the possible values of `$proxy_env_vars` above. One is empty string, and the other has a trailing space.